### ISSUE TITLE
[question] Expected SVF results / Analysis type/settings to use?

### ISSUE THREAD

**Issue by dtzWill** [unsw-corg/PTABen#2]
Hi!

Thanks for all your work on SVF and making PTABen available!

While testing the [ongoing SVF -> LLVM 4.0 changes](https://github.com/unsw-corg/SVF/pull/18) I found these tests very helpful.

Currently it looks like `fi_tests` are the default target, and using latest SVF (LLVM 3.8) it looks like all those tests pass.

Unfortunately it seems the other test folders do not currently pass (when using the default options, at least).

Is this expected behavior, or is something wrong on my end?

For things like flow-sensitivity and context-sensitivity I can imagine different analyses are appropriate, should these test sets be used with different options to `wpa`? (`-fspta`, maybe)?

In short: how do /you/ guys use PTABen for testing SVF? Is it expected that some tests fail (things that have yet to be implemented, etc.)?

**Comment by rockysui** [comment]
Hi Will,

For testing flow-sensitive test cases in folder `fs_tests`, it needs to change the option `-ander` to be `-fspta`. When you run the tests cases in `fs_tests`, all the results analyzed by flow-sensitive analysis should pass (marked in green), and some flow-insensitive (Andersen's analysis) will fail (marked in red) for comparison purpose.

Everything should work (results of targeted analysis should all be in green) before pushing the code to repo.

**Comment by dtzWill** [comment]
Great, thanks for the quick response!

Am I understanding this correctly?

"Does SVF work":
* `fi_tests` should all be green with default `-ander`
* `fs_tests` should all be green with `-fspta` (and some `fi_tests` might fail)

Should the tests in `cs_tests` and `path_tests` be used as well, or are those mostly for comparison/evaluation purposes?

**Comment by rockysui** [comment]
Yes, correct!

cs_tests and path_tests are for SUPA. One can also use it for comparison purpose.
